### PR TITLE
3.3.0 ExistsIn allowPartialNulls flag docs (once 8908 hits)

### PR DESCRIPTION
**Pull request by ionas**
To be merged once https://github.com/cakephp/cakephp/pull/8908 hits.

### CODE REVIEW

File: en/orm/validation.rst [hunk 1]
```diff
@@ -377,6 +377,16 @@ can help provide a nicer user experience. Because of this CakePHP includes an
 The fields to check existence against in the related table must be part of the
 primary key.
 
+.. versionadded:: 3.2.11
+   You may now enforce existsIn to pass on parts of your composite foreign key to be null.
```
Comment (ravage84) [inline en/orm/validation.rst]: ``existIn()``, may be?

File: en/orm/validation.rst [hunk 2]
```diff
@@ -377,6 +377,16 @@ can help provide a nicer user experience. Because of this CakePHP includes an
...
+
+   // The primary composite key of Nodes is (id, site_id).
```
Comment (ravage84) [inline en/orm/validation.rst]: Why Nodes in capital?

File: en/orm/validation.rst [hunk 3]
```diff
@@ -377,6 +377,16 @@ can help provide a nicer user experience. Because of this CakePHP includes an
...
+   // A node may reference a parent node but does not need to (parent_id = null):
+   $rules->add($rules->existsIn(['parent_id', 'site_id'], 'ParentNodes', ['partialNullsPass' => true]));
+
+   // A Node however must in addition also always reference a Site.
```
Comment (ravage84) [inline en/orm/validation.rst]: I guess node and site

File: en/orm/validation.rst (hunk 2 above)
Comment (ionas) [inline en/orm/validation.rst]: because its the Association name (Nodes, ParentNodes) I can change it to nodes. no problem.

**Comment by ionas** [comment]
**I assume 3.next will be 3.2.12 and not 3.3.0**?!

As the feature got merged: https://github.com/cakephp/cakephp/pull/8908#issuecomment-229493732 could this docs update be merged as well?

**Comment by markstory** [comment]
> I assume 3.next will be 3.2.12 and not 3.3.0?!

No `3.next` will turn into 3.3.0

**Comment by ionas** [comment]
Updated `versionadd` and title. Do I need to retarget?

**Comment by markstory** [comment]
Merged into 3.3 in 32907070c948cd068e99daf947465e697f159e1e

**Pull request closed without merging**